frontend/src/pages/landing-page/index.tsx

```diff
@@ -0,0 +1,542 @@
+              </p>
+              <a className={styles.pubArticleLink} href="#">
+                Read More
+                <span className={styles.linkArrow}>
+                  <LinkArrow />
+                </span>
+              </a>
+            </div>
+
+            <div className={styles.pubArticle}>
+              <span className={styles.pubArticleDate}>03.03.21 - medRxiv</span>
+              <h3 className={styles.pubArticleTitle}>
+                Peng J et al.: Estimation of secondary household attack rates
+                for emergent SARS-CoV-2 variants detected by genomic
+                surveillance at a community-based testing site in San Francisco.
+              </h3>
+              <p className={styles.pubArticleCitation}>
+                medRxiv, 2021 doi:10.1101/2021.03.01.21252705
+              </p>
+              <a className={styles.pubArticleLink} href="#">
+                Read More
+                <span className={styles.linkArrow}>
+                  <LinkArrow />
+                </span>
+              </a>
+            </div>
+
+            <div className={styles.pubArticle}>
+              <span className={styles.pubArticleDate}>03.03.21 - medRxiv</span>
+              <h3 className={styles.pubArticleTitle}>
+                Peng J et al.: Estimation of secondary household attack rates
+                for emergent SARS-CoV-2 variants detected by genomic
+                surveillance at a community-based testing site in San Francisco.
+              </h3>
+              <p className={styles.pubArticleCitation}>
+                medRxiv, 2021 doi:10.1101/2021.03.01.21252705
+              </p>
+              <a className={styles.pubArticleLink} href="#">
+                Read More
+                <span className={styles.linkArrow}>
+                  <LinkArrow />
+                </span>
+              </a>
+            </div>
+          </div>
+          <div className={styles.publicationsSeparator}></div>
+          <div className={styles.newsPublications}>
+            <span className={styles.pubSectionTitle}>
+              CELL X GENE IN THE NEWS
+            </span>
+            <div className={styles.pubSectionImage}>
+              <Image src={NewsImage1} alt="" />
+            </div>
+            <div className={styles.pubSectionImage}>
+              <Image src={NewsImage2} alt="" />
+            </div>
+          </div>
+        </div>
+      </main>
+      <footer className={styles.footer}>
+        <div className={styles.footerTopContainer}>
+          <div className={styles.footerLogo}>
+            <Link href="/" passHref>
+              <a>
+                <Image
+                  data-test-id="cellxgene-logo"
+                  src={Logo}
+                  alt="cellxgene logo"
+                />
+              </a>
+            </Link>
+          </div>
+
+          <div className={styles.footerTopLinks}>
+            <a href="https://github.com/chanzuckerberg/aspen/" target="_blank">
```

Comment:
let's add `rel="noopener"` for security!

here and throughout

source: https://web.dev/external-anchors-use-rel-noopener/

Reply:
done!